articles/getting-started-with-jwt-using-angular8-and-nodejs/index.md

```diff
@@ -0,0 +1,362 @@
+
+For this task, I will be using the express-jwt function.
+
+```javascript
+app.get('/path1', (req, res) => {
+    res.status(200)
+        .json({
+            "success": true,
+            "msg": "Secrect Access Granted"
+        });
+});
+```
+
+Using expressJWT function:
+
+```javascript
+//This is to allow access to the path with no token authentication
+app.use(expressJWT({ secret: secret})
+    .unless(
+        { path: [
+            '/token/sign'
+        ]}
+    ));
+```
+
+In the next step, I will be trying to access the path without a token sent in
+the header using the postman tool.
+
+![401 Error](/articles/getting-started-with-jwt-using-angular8-and-nodejs/401-unauthorized.png)
+
+On trial, the app did not allow me to access the path, the error 401
+Unauthorized was displayed as expected. Next will be to test it with the token I
+first obtained from the token/sign route as show below:
+
+```javascript
+app.get('/path1', (req, res) => {
+    res.status(200)
+        .json({
+            "success": true,
+            "msg": "Secret Access Granted"
+        });
+});
+```
+
+On adding the Bearer Token, the success message is displayed as shown below:
+
+![success message](/articles/getting-started-with-jwt-using-angular8-and-nodejs/success-token.png)
+
+Heading back to Angular, I will create a new component home by running the
+following command:
+
+```javascript
+ng generate component home
+```
+
+Modifying my home.component.ts located at *src/app/home/home.component.ts*  as
+below:
+
+```javascript
+signIn() {
+    this.http.get(this.API_URL + '/token/sign')
+      .subscribe(
+        (res) => {
+          console.log(res);
+          if (res['token']) {
+            localStorage.setItem('token', res['token']);
+          }
+        },
+        (err) => {
+          console.log(err);
+        }
+      );    
+  }
+
+  getPath() {
+    this.http.get(this.API_URL + '/path1')    
+      .subscribe(
+        (res) => {
+          console.log(res);
+        },
+        (err) => {
+          console.log(err);
+        }
+      );       
+  }
+
+```
+
+The *signIn()* function in the above code requests a token a stores it into local
+storage, while the second *getPath()* function is for requesting the path.
+
+Coding a simple HTML page to test the two functions above, I will browse to */src/app/home/home.component.html* and code the below: 
+
+```HTML
+<h3>JSON Web Test</h3>
+
+<button (click)="signIn()">Sign In</button>
+<button (click)="getPath()">Get Path 1</button>
+```
+
+To run an Angular project on the browser, I will browse the project folder
+using **cmd** and run the command *ng serve*. The command will compile the whole Angular project and launch the page on the browser. Press F12 on the keyboard to open developer tools console window and then click on the **Sign In** button to get a token as explained above, then click on the **Get Path1** Button to access the path as shown below:
+
```

Comment:
Please rephrase the first sentence to read: "To run the Angular project, browse to the project folder and run the command \`ng serve` on a terminal." 

The second sentense is too long. Consider breaking it down into simpler sentences.